### ISSUE TITLE
Is the linter used?

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/concerto#84]
If I run lint on `concerto-core`:

```
bash-3.2$ npm run lint 

> @accordproject/concerto@0.80.3 lint /Users/jeromesimeon/git/concerto/packages/concerto-core
> eslint .


/Users/jeromesimeon/git/concerto/packages/concerto-core/lib/baseexception.js
  15:1  error  'use strict' is unnecessary inside of modules  strict

/Users/jeromesimeon/git/concerto/packages/concerto-core/lib/basefileexception.js
  15:1  error  'use strict' is unnecessary inside of modules  strict

/Users/jeromesimeon/git/concerto/packages/concerto-core/lib/codegen/filewriter.js
  15:1  error  'use strict' is unnecessary inside of modules  strict

/Users/jeromesimeon/git/concerto/packages/concerto-core/lib/codegen/fromjs/apisignaturegenerator.js
  15:1  error  'use strict' is unnecessary inside of modules  strict

/Users/jeromesimeon/git/concerto/packages/concerto-core/lib/codegen/fromjs/jsongenerator.js
  15:1  error  'use strict' is unnecessary inside of modules  strict

/Users/jeromesimeon/git/concerto/packages/concerto-core/lib/codegen/fromjs/plantumlgenerator.js
   15:1   error    'use strict' is unnecessary inside of modules  strict
   42:13  warning  Unexpected console statement                   no-console
   46:13  warning  Unexpected console statement                   no-console
  105:9   warning  Unexpected console statement                   no-console
  106:9   warning  Unexpected console statement                   no-console
  107:9   warning  Unexpected console statement                   no-console
  110:9   warning  Unexpected console statement                   no-console
...
```

**Comment by jeromesimeon** [comment]
An update: the linter is _not_ used... I'm looking into it.

**Comment by jeromesimeon** [comment]
Fixed in PR #86
